### ISSUE TITLE
Add instructions for setting up ruby and sass

### ISSUE THREAD

**Issue by tomasr8** [proevo/pythondotorg#2493]
I was trying to setup the development environment for another issue I wanted to fix and I ran into problems
regarding setting up ruby and sass. When using a ruby version >=3 the compass version specified in the lockfile fails with the following error:

```
.../.rvm/rubies/ruby-3.0.0/lib/ruby/3.0.0/set/sorted_set.rb:4:in `rescue in <top (required)>':
  The `SortedSet` class has been extracted from the `set` library.
  You must use the `sorted_set` gem or other alternatives. (RuntimeError)
```
(This is fixed by installing compass >=1.0)

Anyway, this wasn't the only issue I had and as someone who's never really used ruby before I think it'd help to have some instructions on how to install the dependencies and perhaps we could also the bump their versions.

**Comment by ned-deily** [comment]
Can you be more specific about in what context this problem arose and what web page needs more information? I'm not aware of a need for ruby or sass when building or using cPython itself. (Update: sorry, I subsequently realized this was in reference to building pythondotorg itself.)

**Comment by tomasr8** [comment]
Indeed, this is about settting up the website, not cpython ;)

**Comment by tomasr8** [comment]
The looks to be resolved by #2623 and #2619 :)
